alicloud/table_alicloud_ecs_image.go

```diff
@@ -22,7 +22,8 @@ func tableAlicloudEcsImage(ctx context.Context) *plugin.Table {
 			Hydrate: listEcsImages,
```

Comment:
Should we include `image_id` in the List key columns for optimization since Get is now having both `image_id` and `region`?